bridge-history-api/internal/controller/history_controller.go

```diff
@@ -28,33 +35,45 @@ func (c *HistoryController) GetAllClaimableTxsByAddr(ctx *gin.Context) {
 		types.RenderFailure(ctx, types.ErrParameterInvalidNo, err)
 		return
 	}
-	offset := (req.Page - 1) * req.PageSize
-	limit := req.PageSize
-	txs, total, err := c.historyLogic.GetClaimableTxsByAddress(ctx, common.HexToAddress(req.Address), offset, limit)
+
+	cacheKey := "claimableTxsByAddr:" + req.Address
```

Comment:
Move cacheKey out from this function as a const value.

Reply:
changed in [850be1e](https://github.com/scroll-tech/scroll/pull/966/commits/850be1e6667441fc8f118dac43603f14a97fd691)